### DISCUSSION TITLE
Fold all code blocks in `.qmd` file when editing in VSCode

### DISCUSSION THREAD

**Discussion by i-thot-u-wanted2dance** [quarto-dev/quarto-cli#5643] [Feature Requests]
It would be great to have a command that folds all code blocks to "get them out of the way" while working on the text of `.qmd` file.  I work in VSCode, so I have in mind a command that would be accessible from the Command Palette.

**Comment by cscheid** [comment]
> I work in VSCode, so I have in mind a command that would be accessible from the Command Palette.

Just FYI, IDE feature requests belong in https://github.com/quarto-dev/quarto.

**Comment by mcanouil** [comment]
FYI, there are already commands/shortcuts to fold code blocks in general, see https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-reference.
On Windows
<img width="317" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/cd0a1c48-3d57-46aa-955d-2dce0e5193b4">

**Reply by i-thot-u-wanted2dance** [reply]
FYI, I know this.  None of them implement the requested feature.
